docs/reference/edot-collector/config/authentication-methods.md

```diff
@@ -0,0 +1,239 @@
+---
+navigation_title: Authentication methods
+description: Learn how to configure authentication for the EDOT Collector using API key authentication, bearer token authentication, and other security methods.
+applies_to:
+  stack: ga
+  serverless: ga
+  product:
+    edot_collector: ga
+products:
+  - id: cloud-serverless
+  - id: observability
+  - id: edot-collector
+---
+
+# Authentication methods for the EDOT Collector
+
+The {{edot}} Collector supports multiple authentication methods to secure connections and ensure only authorized clients can send telemetry data. This guide covers the available authentication extensions and how to configure them.
+
+## Overview
+
+Authentication in the EDOT Collector is handled through extensions that implement the `extensionauth` interface. These extensions can be used to:
+
+- Authenticate incoming requests from SDKs and other collectors.
+- Authenticate outgoing requests to external services.
+- Secure internal Collector communications.
+
+## Available authentication extensions
+
+The EDOT Collector supports the following authentication extensions:
+
+### Elastic API Key Authentication (`apikeyauth`)
+
+The `apikeyauth` extension is an Elastic-specific authentication method that validates Elasticsearch API keys against your {{es}} cluster. This extension is ideal for authenticating requests from EDOT SDKs and other Collectors that use Elasticsearch API keys.
+
+### Bearer Token Authentication (`bearertokenauth`)
+
+The `bearertokenauth` extension is an upstream OpenTelemetry authentication method that supports static bearer tokens. This extension is useful for token-based authentication scenarios.
+
+## Configuration examples
+
+These examples show how to configure the `apikeyauth` and `bearertokenauth` extensions.
+
+### Elastic API key authentication
+
+Configure the `apikeyauth` extension to authenticate incoming requests:
+
+```yaml subs=true
+extensions:
+  apikeyauth:
+    endpoint: "https://example.com:9200"
+    application_privileges:
+      - application: "apm"
+        privileges: ["config_agent:read"]
+        resources: ["*"]
+    cache:
+      capacity: 1000
+      ttl: "5m"
+      pbkdf2_iterations: 10000
+      key_headers: ["X-Tenant-Id"]
+
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        auth:
+          authenticator: apikeyauth
+      http:
+        auth:
+          authenticator: apikeyauth
+
+service:
+  extensions: [apikeyauth]
+```
+
+#### Configuration options
+
+The following configuration options are available for the `apikeyauth` extension:
+
+| Option | Type | Description | Default |
+|--------|------|-------------|---------|
+| `endpoint` | string | The Elasticsearch endpoint for API key validation | Required |
+| `application_privileges` | array | List of required application privileges and resources | Required |
```

Comment:
Since you specify the `cache.*`, maybe we can also add `application_privileges.*`, what do you think?

https://github.com/elastic/opentelemetry-collector-components/blob/07718e71c16f587bc55e43036b16256f6a6a90d1/extension/apikeyauthextension/config.go#L40-L52

So maybe add this:

- `application_privileges.application`: (optional) name of the application for which privileges are defined
- `application_privileges.privileges`: (optional) list of application-specific privileges that the API Key must have to be considered valid.
- `application_privileges.resources`: (optional) list of application-specific resources that the API Key must have access to be considered valid.